modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStore.java

```diff
@@ -285,27 +280,46 @@ public CompletableFuture<DeltaFilePageStoreIo> getOrCreateNewDeltaFile(
             return newDeltaFilePageStoreIoFuture;
         }
 
-        int nextIndex = deltaFilePageStoreIos.isEmpty() ? 0 : deltaFilePageStoreIos.get(0).fileIndex() + 1;
+        DeltaFilePageStoreIo newDeltaFilePageStoreIo = addNewDeltaFilePageStoreIoToHead(pageIndexesSupplier.get(), deltaFilPathFunction);
```

Comment:
Please rename `deltaFilPathFunction`, there's a letter missing

Reply:
What name do you suggest?

Reply:
deltaFil**e**PathFunction
I said, there's a letter missing

Reply:
I saw it and corrected it, thanks!